03-Mains/Temperature_Salinity_Diagram.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "65fe6a4e-f751-40c3-80ed-12c66e728784",
```

Comment:
<p>Do we need to wrap this in a function? The notebook would be simpler without this additional layer of abstraction.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/619/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>